src/llama_stack/providers/utils/inference/openai_compat.py

```diff
@@ -957,23 +961,24 @@ def openai_messages_to_messages(
     """
     Convert a list of OpenAIChatCompletionMessage into a list of Message.
     """
-    converted_messages = []
+    converted_messages: list[Message] = []
     for message in messages:
+        converted_message: Message
         if message.role == "system":
-            converted_message = SystemMessage(content=openai_content_to_content(message.content))
+            converted_message = SystemMessage(content=openai_content_to_content(message.content))  # type: ignore[arg-type]  # OpenAI content union broader than Message content union
```

Comment:
I don't understand why this is. should we update `openai_content_to_content` type signature or what? or is it the message.content access that's causing this?